src/public/parts/shells/chat/public/src/templates/group/settings/channel_permissions_panel.html

```
@@ -16,42 +16,33 @@ <h2 class="card-title" data-i18n="chat.group.settings.page.tabs.channelPermissio
-									</div>
+			<div class="flex-1 min-w-0 space-y-3">
+				${rolePanels.map(panel => `
+				<details class="settings-role" data-role-panel="${escapeHtml(panel.roleId)}" ${panel.open ? 'open' : ''}>
+					<summary>
+						<span class="settings-role-swatch" style="background-color: ${escapeHtml(panel.color)}" aria-hidden="true"></span>
+						<span class="truncate">${escapeHtml(panel.name)}</span>
+						${panel.hasOverride ? '<span class="badge badge-sm badge-warning" data-i18n="chat.group.settings.page.channelPerms.overridden"></span>' : ''}
+					</summary>
+					<div class="settings-role-body">
+						<div class="settings-role-perms">
+							${panel.permRows.map(row => `
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)